bin/kafka-run-class.sh

```diff
@@ -20,6 +20,15 @@ then
   exit 1
 fi
 
+check_file() {
```

Comment:
I think the name could be more clear on whats being checked. Maybe `isTestJar` or something like that.